tests/device_tests/bitcoin/test_signtx.py

```diff
@@ -392,17 +386,18 @@ def test_two_two(client):
                 request_output(0),
                 messages.ButtonRequest(code=B.ConfirmOutput),
                 request_output(1),
+                messages.ButtonRequest(code=B.ConfirmOutput),
```

Comment:
This is always a problem. Expected responses **must** stay identical except for txhash references.

The problem here is that your "change" output is not change -- it's from a different account. In this case you can fix it by switching the first output to change and the second to address.

Reply:
Correction: expected responses can of course change if the prevtx changes, or if you deliberately change the number of outputs/inputs, or similar.

But the number of confirmation _should_ stay the same, except in maybe some exceptional cases.

Reply:
Changed in eaaa13742bdbe50f06a8452d8f6a93a52365a211 - so that it recognizes change address